packages/camera/camera_android_camerax/android/build.gradle

```diff
@@ -60,7 +60,7 @@ android {
     lintOptions {
         checkAllWarnings true
         warningsAsErrors true
-        disable 'AndroidGradlePluginVersion', 'GradleDependency', 'InvalidPackage', 'NewerVersionAvailable'
+        disable 'AndroidGradlePluginVersion', 'GradleDependency', 'InvalidPackage', 'NewerVersionAvailable', 'UnsafeOptInUsageError'
```

Comment:
In https://github.com/flutter/packages/pull/8618, I manually added this to the generated pigeon file `CameraXLibrary.g.kt`. To prevent the need to do manually change generated code, I moved the disable here.

Reply:
Working on another camera PR (https://github.com/flutter/packages/pull/9264) and I created a baseline lint file for that same `UnsafeOptInUsageError` lint. I think that might be better just in case the lint is valid for non-generated files, but let me know what you think!

If you agree and these other lints are from `CameraXLibrary.g.kt`, maybe we should remove them and just regenerate the baseline file.

Reply:
I'll wait for https://github.com/flutter/packages/pull/9264 to land and then regen the baseline file for this PR